src/api/api.ts

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+
+const app = express();
+const port = process.env.EXPRESS_PORT;
+
+app.get('/coverage', (req, res) => {
+  if (port === process.env.EXPRESS_PORT) {
+    res.json((global as any).__coverage__).send();
+  } else {
+    res.sendStatus(500);
```

Comment:
Line 10 here is not covered. The coverage report suggests that line 14 is not covered.